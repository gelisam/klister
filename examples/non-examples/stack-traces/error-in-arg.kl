#lang "prelude.kl"

(define fail (lambda (thing) (+ thing )))

(example (fail (error 'badd)))
