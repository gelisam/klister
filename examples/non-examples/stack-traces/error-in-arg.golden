In Phase: p0:
    error-in-arg.kl:5.24-5.27: bad
    stack trace:
    ---- with function #<thing>
    ---- Halt
